cmd/tuf/server/main.go

```diff
@@ -81,7 +82,22 @@ func main() {
 			if err != nil {
 				logging.FromContext(ctx).Fatalf("failed to read file %s/%s: %v", fileName, err)
 			}
-			files[file.Name()] = fileBytes
+			// If it's a TSA file, we need to split it into multiple TUF
+			// targets.
+			if strings.Contains(file.Name(), "tsa") {
+				logging.FromContext(ctx).Infof("Splitting TSA certchain into individual certs")
+
+				certFiles, err := certs.SplitCertChain(fileBytes, "tsa")
```

Comment:
To confirm, it's a single file because this is coming from the /certchain endpoint?

Reply:
The createcertchain job creates the entire chain and stuffs it into a single secret. Also, was modeling it after retrieving it from there like that, but since the tsa server takes the chain, that's what I stuff into the secret and just mount it here as well.